websocket/frontend/src/todolist/Todoform.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+class TodoForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      api_url: props.api_url,
+      task: "",
+    }
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleTaskChange = this.handleTaskChange.bind(this);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.formSubmit(event.target)
+  }
+
+  async formSubmit(formData) {
+    let data = new FormData(formData);
+    await fetch(this.state.api_url, {
+      method: 'POST',
+      mode: 'cors',
+      body: data,
+    }).then(response => response.json())
+    .then(response => this.props.updateTodoList(response))
+  }
```

Comment:
```suggestion
  async formSubmit(formData) {
    const data = new FormData(formData);
    const obj = {
      method: 'POST',
      mode: 'cors',
      body: data,
    }
    const res = await fetch(this.state.api_url, obj)
    this.props.updateTodoList(res.json())
  }
```
The above should work